src/com/sales/tax/SalesItem.java

```diff
@@ -0,0 +1,58 @@
+package com.sales.tax;
+
+public class SalesItem {
+    private final String itemName;
+    private final double itemPrice;
+    private final boolean isExempted;
+    private final boolean isImported;
+
+    // Constructor to create the sale item
+    public SalesItem(String itemName, double itemPrice, boolean isExempted, boolean isImported) {
+        this.itemName = itemName;
+        this.itemPrice = itemPrice;
+        this.isExempted = isExempted;
+        this.isImported = isImported;
+    }
+
+    public String getItemName() {
+        return itemName;
+    }
+
+    public double getItemPrice() {
+        return itemPrice;
+    }
+
+    public boolean isExempted() {
+        return isExempted;
+    }
+
+    public boolean isImported() {
+        return isImported;
+    }
+
+    //Method to calculate the tax depending on the criteria (exempted and imported)
+    public TaxAndPrice calculatePrice(int qty){
+        double totalPrice=0;
+        double tax =0;
+        double exemptedTaxRate = (10.0 / 100);
+        double importedTaxRate = (5.0 / 100);
+        if ((this.isExempted) && (this.isImported)) {
```

Comment:
This bunch of if/elseIf could be simplified with 2 independent if, one for imported one for exempt.